app/serializers/book_serializer.rb

```diff
@@ -0,0 +1,15 @@
+class BookSerializer < ActiveModel::Serializer
+  attributes  :id,
+              :title,
+              :author,
+              :genre,
+              :publisher,
+              :year,
+              :available,
+              :image_url
+
+  def available
+    rent = object.rents.where(to: object.rents.select('MAX(rents.to)')).first
```

Comment:
Maybe this could look better if you used `maximum(:column_name)`

Reply:
I'll try, but I don't understand what you mean

Reply:
it could be `rent = object.rents.where(to: object.rents.maximum(:to)).first`

this gives you the rent with the maximum end date, so you can check if it exceeds today's date

Reply:
Done